includes/admin/class.llms.admin.builder.php

```diff
@@ -165,8 +165,9 @@ private static function get_custom_schemas() {
 	 *
 	 * @since 3.14.8
 	 * @since 3.16.12 Unknown.
+	 * @since [version] Allow LMS managers to get all lessons. {@link https://github.com/gocodebox/lifterlms/issues/1849}.
 	 *
-	 * @param int    $course_id   WP Post ID of the course
+	 * @param int    $course_id   NOT USED. WP Post ID of the course, lesson or llms_quiz.
```

Comment:
since this is a private method, can't we just get rid of this parameter?

Reply:
@eri-trabiccolo,
I think you are right. It looks like the `$course_id` has never been used and is a mistake left in when `get_orphaned_lessons()` was converted to `get_existing_lessons()` in 2be557fa20ae358ee137e023dce82ff4219cdb4e.